src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks.js

```diff
@@ -0,0 +1,35 @@
+import { clone, pathOr } from 'ramda';
+import convertParagraph from './convertParagraph';
+
+const handleMissingType = block =>
+  console.log(`Missing type field on block ${block.type}`); // eslint-disable-line no-console
+
+const typesToConvert = {
+  paragraph: convertParagraph,
+};
+
+const parseBlockByType = block => {
+  const { type } = block;
```

Comment:
Small risk here with janky data - blocks array having an undefined entry will throw an exception

Reply:
Should now be covered by the `if` on line 12